### PR TITLE
LPS-190401

### PR DESCRIPTION
**Pull request by TamyrisTorres**
Tickets: https://liferay.atlassian.net/browse/LPS-190401
https://liferay.atlassian.net/browse/LPS-188828

### CODE REVIEW

**Comment by TamyrisTorres** [comment]
ci:test:sf

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaLayoutServicesCheck.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+				javaMethodContent, javaMethodContent,
+				ArrayUtil.toStringArray(parameterList), parameterTypes)) {
+
+			addMessage(
+				fileName,
+				StringBundler.concat(
+					"Unable to format methods addLayout and updateLayout from ",
+					"LayoutService, LayoutLocalService, LayoutServiceUtil and ",
+					"LayoutLocalServiceUtil fill the new parameters manually, ",
+					"see LPS-188828 and LPS-190401"));
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaLayoutServicesCheck.java:40]: @TamyrisTorres, can we adjust only the final part of this message?
```
... LayoutLocalServiceUtil. Fill the new parameters, manually ...
```

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaSearchVocabulariesMethodCheck.java [hunk 2]
```diff
@@ -31,64 +35,72 @@ protected String doProcess(
...
-			newContent = StringUtil.replace(newContent, methodCall, newMethod);
+					addMessage(
+						fileName,
+						StringBundler.concat(
+							"Could not resolve types of searchVocabularies ",
+							"method. The method signature has changed to ",
+							"searchVocabularies(long companyId, long[] ",
+							"groupIds, String title, int[] visibilityTypes, ",
+							"int start, int end, Sort sort). Fill the new ",
+							"parameters manually."));
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaSearchVocabulariesMethodCheck.java:90]: @TamyrisTorres, can we rephrase this message as was done in https://github.com/natocesarrego/liferay-portal/pull/1343#discussion_r1290079410?

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaLayoutServicesCheck.testjava [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+		_layoutService.updateLayout(
+			groupId, privateLayout, layoutId, parentLayoutId, nameMap, titleMap, descriptionMap, keywordsMap, robotsMap, type, hidden, friendlyURLMap, hasIconImage, iconBytes, 0, 0, 0, serviceContext);
+
+		// Test LayoutServiceUtil
+
+		LayoutServiceUtil.updateLayout(
+			groupId, privateLayout, layoutId, parentLayoutId, nameMap, titleMap, descriptionMap, keywordsMap, robotsMap, type, hidden, friendlyURLMap, hasIconImage, iconBytes, 0, 0, 0, serviceContext);
+
+		// Test scenario when it is not possible to validate parameter types
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaLayoutServicesCheck.testjava:65]: Nice job, @TamyrisTorres! :smile:

**Review by natocesarrego — COMMENTED** [review]
Good job, Tamyris! Please see my tiny comments. :wink:

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Review by natocesarrego — APPROVED** [review]

**Comment by natocesarrego** [comment]
Pull request approved, @TamyrisTorres!

**Comment by TamyrisTorres** [comment]
ci:forward

**Comment by natocesarrego** [comment]
ci:forward:force

**Pull request closed without merging**